samples-check/src/test/groovy/org/gradle/exemplar/test/runner/SamplesRunnerSadDayIntegrationTest.groovy

```diff
@@ -26,14 +26,14 @@ class SamplesRunnerSadDayIntegrationTest extends Specification {
         notifier.failures[0].description == notifier.tests[0]
 
         def expectedOutput = """
-            Expected sample invocation to succeed but it failed.
-            Command was: 'bash broken'
-            Working directory: '.+/_broken-command.sample'
-            \\[BEGIN OUTPUT\\]
-            bash: broken: No such file or directory
-            
-            \\[END OUTPUT\\]
-        """.stripIndent().trim()
```

Comment:
`stripIndent(true)` would have restored the old behavior

Reply:
TIL 👍 
fixed in 66adad6